Makefile

```
@@ -10,9 +10,16 @@ DOCKER_CACHE       := $(CURDIR)/.cache
 ANTCTL_BINARY_NAME ?= antctl
 OVS_VERSION        := $(shell head -n 1 build/images/deps/ovs-version)
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
+CNI_BINARIES_VERSION := $(shell head -n 1 build/images/deps/cni-binaries-version)
+NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
+WIN_BUILD_TAG      := $(shell echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION|md5sum|head -c 10)
```

```suggestion
WIN_BUILD_TAG      := $(shell echo $(GO_VERSION) $(CNI_BINARIES_VERSION) $(NANOSERVER_VERSION)|md5sum|head -c 10)
```
Otherwise the variables are not rendered, which caused mismatched md5sum

---

Thanks a lot! The a43c77d755 should be the correct hash tag for base-windows image.